### DISCUSSION TITLE
Make global variables bold

### DISCUSSION THREAD

**Discussion by Kamilcuk** [dasupradyumna/midnight.nvim#12] [Ideas]
I think global variables are really important in  a project and midnight does not use any bold fonts.

I propose to make global variables - `@lsp.typemod.variable.globalScope` and `@lsp.mod.globalScope` - bold. Or constants - `@constant` - bold.

Thanks.

**Comment by dasupradyumna** [comment]
That was done on purpose. Users are free to add minor overrides like this using the plugin `setup()` function.
It would be impossible to cater to everyone's personal preferences.

Thank you for using the colorscheme!
